src/wp-includes/functions.php

```diff
@@ -2074,6 +2075,16 @@ function wp_mkdir_p( $target ) {
 		$dir_perms = 0777;
 	}
 
+	/**
+	 * Fires before the directory is created
```

Comment:
```suggestion
	 * Fires before the directory is created.
```
- Docblock subjects should end with a period.